samples/ck-rx65n/basic-sample/src/iotconnect_app.c

```diff
@@ -167,6 +178,63 @@ static void publish_telemetry() {
     }
 }
 
+
+#ifdef SYMMETRIC_KEY_INPUT
+
+#define CREDENTIALS_BLOCK_DATA FLASH_DF_BLOCK_24
+
+
+// be careful with changing these constants
+#define IOTC_CONFIG_BUFF_LEN_GENERIC 64
+#define IOTC_CONFIG_BUFF_LEN_SYMMKEY 256
+typedef struct credentials {
```

Comment:
typedef struct credentials -> typedef struct credentials_t

Then you can use the word "credentials" for variables :-)